### PR TITLE
Show the last-used browser's icon

### PR DESCRIPTION
**Pull request by hannesschulze**
Implementing #41 
~~I also tried to use linked buttons but that didn't quite fit the overall style of the browser imo~~ Added STYLE_CLASS_LINKED to the buttons although that doesn't make a real difference right now

screenshot von 2019-01-12 19 31 42

### CODE REVIEW

**Comment by cassidyjames** [comment]
@hannesschulze thanks for this! I will take a look.

**Comment by cassidyjames** [comment]
Something else I noticed: if you uninstall a browser while Ephemeral is open, it kind of goes crazy. That's not strictly new in this branch, but it's made more obvious if you choose the uninstalled browser as the default one.

**Comment by hannesschulze** [comment]
Maybe it would work if you just watch the desktop file and remove the entry if it gets removed - unfortunately, I don't know how to watch for new browsers on the system.

Should I try to implement that on this branch or should I create another branch and PR?

File: data/gresource.xml [hunk 1]
```diff
@@ -3,5 +3,8 @@
   <gresource prefix="/com/github/cassidyjames/ephemeral">
     <file alias="Application.css">Application.css</file>
   </gresource>
+  <gresource prefix="/com/github/cassidyjames/ephemeral/images">
+    <file alias="arrow-down.svg">images/arrow-down.svg</file>
```
[the remark this answers is not in the document]
Comment (hannesschulze) [reply data/gresource.xml]: I didn't see that icon, thank you!
But I think 16px looks a little bit too big imo:
screenshot von 2019-01-14 17 26 03

**Comment by cassidyjames** [comment]
Playing with some CSS for the linked buttons:

image

**Comment by cassidyjames** [comment]
@hannesschulze thanks for this, I'm going to merge it into a working branch here and continue working on it.

**Pull request merged**